compiler/test/fail_compilation/fail24389.c

```diff
@@ -1,10 +0,0 @@
-/*
-TEST_OUTPUT:
----
-fail_compilation/fail24389.c(10): Error: Gnu Asm not supported - compile this file with gcc or clang
----
-*/
-typedef unsigned long size_t;
-void __qsort_r_compat(void *, size_t, size_t, void *,
-     int (*)(void *, const void *, const void *));
-__asm__(".symver " "__qsort_r_compat" ", " "qsort_r" "@" "FBSD_1.0");
```

Comment:
The test is still useful, as it's checking whether top-level asm declarations are supported, rather than qsort itself (GDC does support top-level asm declarations though in importC).